api/src/main/java/org/pmiops/workbench/db/dao/WorkspaceServiceImpl.java

```diff
@@ -30,7 +27,12 @@
 public class WorkspaceServiceImpl implements WorkspaceService {
   private static final Logger log = Logger.getLogger(WorkspaceService.class.getName());
 
-  @Autowired private WorkspaceDao workspaceDao;
```

Comment:
So the reason we had @Autowired here before was that due to some bug in Spring (which was perhaps fixed in a more recent version of it than we were using), it wasn't able to construct a service implementation with a non-default constructor. @blrubenstein 

Does this code work as expected when you run up the server?

Reply:
Ugh, right you are, I find this form of injection especially distasteful but it's not important for this change. Found what appears to be the causative issue you're describing: https://jira.spring.io/browse/SPR-15600, should be fixed if we were to upgrade our Spring boot version I'd guess (not sure which version).

Anyways, reverting this for now and rebasing.. going to take some effort to figure out how to factor in the `FirecloudService`. Will push and reply once that's ready.

Reply:
OK. 

Feel free to try updating Spring Boot and seeing if it fixes this -- we had meant to do this and just hadn't gotten around to it yet.

Reply:
Sure, I can try and tackle that separately since it's independent: RW-256. This was just a style pet-peeve not needed for this PR.